css/build.css

```diff
@@ -144,4 +144,8 @@
 /* For native devices */
 .native .fa-times-thin {
   padding-top: 4px;
-} 
\ No newline at end of file
+}
+
+.mode-interact .fl-widget-instance {
```

Comment:
I believe this would be applying a minimum height to all instances of all widgets. That's way too broad as a selector.

Reply:
@tonytlwu I've added new css class and rule for min-height.